content/sensu-go/5.11/installation/install-sensu.md

```diff
@@ -5,385 +5,330 @@ description: "Sensu Go is available for Linux, Windows (agent and CLI only), mac
+### Install
 
-### 1. Install the package
+The Sensu backend is available for Ubuntu/Debian, RHEL/CentOS, and Docker.
+See the [supported platforms page][5] for more information.
 
-{{< platformBlockClose >}}
+##### 1. Download
 
-{{< platformBlock "Ubuntu/Debian" >}}
+{{< language-toggle >}}
 
-#### Ubuntu/Debian
+{{< highlight Docker >}}
+# All Sensu Docker images contain a Sensu backend and a Sensu agent
 
-Add the Sensu repository.
+# Pull the Alpine-based image
+docker pull sensu/sensu
 
-{{< highlight shell >}}
-curl -s https://packagecloud.io/install/repositories/sensu/stable/script.deb.sh | sudo bash
+# Pull the image based on Red Hat Enterprise Linux
+docker pull sensu/sensu-rhel
 {{< /highlight >}}
 
-Install the `sensu-go-agent` package.
+{{< highlight "Ubuntu/Debian" >}}
+# Add the Sensu repository
+curl -s https://packagecloud.io/install/repositories/sensu/stable/script.deb.sh | sudo bash
 
-{{< highlight shell >}}
-sudo apt-get install sensu-go-agent
+# Install the sensu-go-backend package
+sudo apt-get install sensu-go-backend
 {{< /highlight >}}
 
-{{< platformBlockClose >}}
-
-{{< platformBlock "RHEL/CentOS" >}}
-#### RHEL/CentOS
-
-Add the Sensu repository.
-
-{{< highlight shell >}}
+{{< highlight "RHEL/CentOS" >}}
+# Add the Sensu repository
 curl -s https://packagecloud.io/install/repositories/sensu/stable/script.rpm.sh | sudo bash
-{{< /highlight >}}
-
-Install the `sensu-go-agent` package.
 
-{{< highlight shell >}}
-sudo yum install sensu-go-agent
+# Install the sensu-go-backend package
+sudo yum install sensu-go-backend
 {{< /highlight >}}
 
-{{< platformBlockClose >}}
-
-{{< platformBlock "Windows" >}}
+{{< /language-toggle >}}
 
-#### Windows {#windows-agent}
+##### 2. Configure and start
 
-Download the Sensu agent for Windows [`amd64`](https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.11.1/sensu-go-agent_5.11.1.5015_en-US.x64.msi) or [`386`](https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.11.1/sensu-go-agent_5.11.1_5015_en-US.x86.msi) architectures.
+You can configure Sensu using `sensu-backend start` flags or an /etc/sensu/backend.yml file, the former taking precedence.
+At a minimum, the Sensu backend requires the `state-dir` flag, but here are some other useful configs and templates.
 
-{{< highlight text >}}
-Invoke-WebRequest https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.11.1/sensu-go-agent_5.11.1.5015_en-US.x64.msi  -OutFile "$env:userprofile\sensu-go-agent_5.11.1.5015_en-US.x64.msi"
-{{< /highlight >}}
+{{< language-toggle >}}
 
-Start the installation wizard.
-
-{{< highlight text >}}
-msiexec.exe /i $env:userprofile\sensu-go-agent_5.11.1.5015_en-US.x64.msi
+{{< highlight Docker >}}
+docker run -v /var/lib/sensu:/var/lib/sensu -d --name sensu-backend -p 2380:2380 -p 3000:3000 -p 8080:8080 -p 8081:8081 sensu/sensu:latest sensu-backend start
```

Comment:
You can skip the volume mount here. It's really only if you're planning on needing to have cached assets & such. Useful for demos, less so in prod, I'd imagine. Otherwise, LGTM

Reply:
@cwjohnston recommended that we keep the volume mount here, but we can always revisit.